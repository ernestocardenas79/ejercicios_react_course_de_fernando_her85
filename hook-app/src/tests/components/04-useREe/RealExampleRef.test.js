import React from 'react';
import { shallow } from 'enzyme';
import { MultipleCustomHooks } from '../../../components/03-examples/MultipleCustomHooks';
import { RealExampleRef } from '../../../components/04-useRef/RealExampleRef';
describe('Pruebas en <RealExampleREf />', () => {
  const wrapper = shallow(<RealExampleRef />);
  test('debe de mostrarse correctamente', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
  });

  test('debe de mostrar el componente <MultipleCustomHooks />', () => {
    wrapper.find('button').simulate('click');
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
  });
});
