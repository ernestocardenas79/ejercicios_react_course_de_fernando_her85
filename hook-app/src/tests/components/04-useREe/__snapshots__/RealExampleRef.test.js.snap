// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <RealExampleREf /> debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
